etc/conda-install.sh

```diff
@@ -114,6 +110,7 @@ elif [ -n "${CHANGED// }" ]; then
 fi
 
 if [ "$RECREATE_CONDA_ENV" -eq "1" ]; then
+    conda remove -n base mamba
     mamba env remove --name $ENV_NAME
```

Comment:
How is the `mamba env remove` command supposed to run if the package was removed?

Reply:
Good catch - will fix now